include/gfx.h

```diff
@@ -0,0 +1,204 @@
+#ifndef GFX_H
+#define GFX_H
+
+#include "ultra64.h"
+#include "PR/sched.h"
+#include "thga.h"
+#include "alignment.h"
+
+
+struct GameState;
+
+typedef struct {
+    /* 0x000 */ Gfx taskStart[9];
+    /* 0x048 */ Gfx clearZBuffer[8];       // original name: clear_zb_dl
+    /* 0x088 */ Gfx clearFrameBuffer[5];   // original name: clear_fb_dl
+    /* 0x0B0 */ Gfx setupBuffers[6];       // setup framebuffer and zbuffer
+    /* 0x0E0 */ Gfx unk_E0[12];            // unused
+    /* 0x140 */ Gfx syncSegments[17];
+    /* 0x1C8 */ Gfx setScissor[2];
+    /* 0x1D8 */ Gfx unk_1D8[25];           // unused
+    /* 0x2A0 */ Gfx disps[5];
+    /* 0x2C8 */ Gfx clearFillRect[3];      // fillrect for clearing buffers
+    /* 0x2E0 */ Gfx fillRect[3];           // fillrect for general purpose
+    /* 0x2F8 */ Gfx debugDisp[1];
+} GfxMasterList; // size = 0x300
+
+#define GFXPOOL_HEAD_MAGIC 0x1234
+#define GFXPOOL_TAIL_MAGIC 0x5678
+
+typedef struct GfxPool {
+    /* 0x00000 */ u16 headMagic; // GFXPOOL_HEAD_MAGIC
+    /* 0x00008 */ GfxMasterList master;
+    /* 0x00308 */ Gfx polyXluBuffer[0x800];
+    /* 0x04308 */ Gfx overlayBuffer[0x400];
+    /* 0x06308 */ Gfx workBuffer[0x40];
+    /* 0x06508 */ Gfx debugBuffer[0x40];
+    /* 0x06708 */ Gfx polyOpaBuffer[0x3380];
+    /* 0x20308 */ u16 tailMagic; // GFXPOOL_TAIL_MAGIC
+} GfxPool; // size = 0x20310
+
+typedef struct GraphicsContext {
+    /* 0x000 */ Gfx*        polyOpaBuffer;  // Pointer to "Zelda 0"
+    /* 0x004 */ Gfx*        polyXluBuffer;  // Pointer to "Zelda 1"
+    /* 0x008 */ char        unk_8[0x8];     // Unused, could this be pointers to "Zelda 2" / "Zelda 3"
+    /* 0x010 */ Gfx*        overlayBuffer;  // Pointer to "Zelda 4"
+    /* 0x014 */ u32         unk_14;
+    /* 0x018 */ char        unk_18[0x20];
+    /* 0x038 */ OSMesg      msgBuff[8];
+    /* 0x058 */ OSMesgQueue* schedMsgQ;
+    /* 0x05C */ OSMesgQueue queue;
+    /* 0x074 */ char        unk_74[0x04];
+    /* 0x078 */ OSScTask    task;
+    /* 0x0E0 */ char        unk_E0[0xD0];
+    /* 0x1B0 */ Gfx*        workBuffer;
+    /* 0x1B4 */ TwoHeadGfxArena work;
+    /* 0x1C4 */ Gfx*        debugBuffer;
+    /* 0x1C8 */ TwoHeadGfxArena debug;
+    /* 0x1D8 */ char        unk_1D8[0xAC];
```

Comment:
```suggestion
    /* 0x1D8 */ UNK_TYPE1 unk_1D8[0xAC];
```
These seem unused? Applies to a few. Could always do this cleanup later though